master/buildbot/util/service.py

```diff
@@ -189,19 +191,39 @@ def getConfigDict(self):
                 'args': self._config_args,
                 'kwargs': self._config_kwargs}
 
+    @defer.inlineCallbacks
     def reconfigServiceWithSibling(self, sibling):
         # only reconfigure if sibling is configured differently.
         # sibling == self is using ComparableMixin's implementation
         # only compare compare_attrs
         if self.configured and sibling == self:
-            return defer.succeed(None)
+            defer.returnValue(None)
         self.configured = True
-        return self.reconfigService(*sibling._config_args,
-                                    **sibling._config_kwargs)
+        # render renderables in parallel
+        # Properties import to resolve cyclic import issue
+        from buildbot.process.properties import Properties
+        p = Properties()
+        p.master = self.master
+        # render renderables in parallel
+        renderables = []
+        kwargs = {}
+        accumulateClassList(self.__class__, 'renderables', renderables)
```

Comment:
why renderables? secrets!

There are some reporters that allows renderable to be configured, and which will render them for each builds. So if they are already rendered at reconfig, the feature will break. That is strange that the unit tests didn't catch this.